aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -45,12 +46,23 @@ class QAdd final : public c10::OperatorKernel {
     TORCH_CHECK(qa.scalar_type() == qb.scalar_type(),
                 "Add operands should have same data type.");
     auto qc = at::_empty_affine_quantized(qa.sizes(),
-      at::device(kCPU).dtype(qa.scalar_type()),
-      scale, zero_point);
+      at::device(kCPU).dtype(qa.scalar_type()), scale, zero_point);
     return _add_out<ReLUFused>(qc, qa, qb);
   }
 };
 
+template <bool ReLUFused = false>
+class QAddOut final : public c10::OperatorKernel {
+ public:
+  Tensor operator()(at::Tensor qa, at::Tensor qb, at::Tensor out) {
+    TORCH_CHECK(qa.numel() == qb.numel(),
+                "Add operands must be the same size!");
+    TORCH_CHECK(qa.scalar_type() == qb.scalar_type(),
+                "Add operands should have same data type.");
+    return _add_out<ReLUFused>(out, qa, qb);
```

Comment:
Please add a check for qscheme. We can mix and match per_tensor_affine and per_tensor_symmetric, but per_channel is not supported.

Reply:
Resolved in the previous diff